server/application-server/src/main/java/de/tum/cit/aet/helios/config/SecurityConfig.java

```diff
@@ -0,0 +1,42 @@
+package de.tum.cit.aet.helios.config;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebSecurity
+public class SecurityConfig {
```

Comment:
@egekocabas I added some simple security config because I was getting "No 'Access-Control-Allow-Origin' header is present on the requested resource." error